app/sessions/session-list/session-list.component.ts

```diff
@@ -46,12 +46,12 @@ export class SessionListComponent implements OnInit {
     public ngOnInit() {
         this._data.currentMessage.subscribe(message => this.message = message);
         this._sessionsService.items.subscribe((observer) => {
-            let delay = 0;
-            observer.forEach((value: SessionModel, i: number, array: Array<SessionModel>) => {
-                delay = delay + 500;
-                setTimeout(() => {
-                    value.triggerShow.next(true);
-                }, delay);
+        let delay = 0;
+        observer.forEach((value: SessionModel, i: number, array: Array<SessionModel>) => {
+            // console.log("Within itemes.subscribe: " + value.sessionTitle);
+            this._zone.run(() => {
+                value.triggerShow.next(true);
```

Comment:
Did this solve the reloading issue for session-list on initial launch?

Reply:
No :( Just didn't make any sense to have those timeouts so removed it.

Reply:
Oh okay no problem